site/accessibility.html

```diff
@@ -0,0 +1,105 @@
+{# ---
```

Comment:
Why keep this?

Reply:
We are leaving this as a reminder, we will add a way to have this functionality later.